parts/22-query.sh

```diff
@@ -1431,45 +1431,64 @@ query_tool-memory-per-inputs() { ##? <tool_id> [--like]: See memory usage and in
 	EOF
 }
 
-query_monthly-cpu-stats() { ##? [year] : CPU years/hours allocated to tools by month
+query_monthly-cpu-stats() { ##? [--nb_users] [--filter_email=<domain>] [year]: CPU years/hours allocated to tools by month (+ nb of users)
```

Comment:
```suggestion
query_monthly-cpu-stats() { ##? [--nb-users] [--filter-email=<domain>] [year]: CPU years/hours allocated to tools by month (+ nb of users)
```

just to reduce the times i have to press <kbd>shift</kbd> :) the variable will still be with underscores of course